File: C - Problem Sets/mario - more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    char block = '#';
    char space  = ' ';
    int intInput;
    while(true){
        printf("Please enter a number: \n");
        intInput = get_int();
        if (intInput >= 0 && intInput <= 23){
            break;
        }
        else{
            continue;
        }
    }

    for (int x = 0; x < intInput; x++){
        
        for (int t = 1; t < (intInput - x); t++){
            printf("%c", space);
        }
        for (int y = 0; y < x + 1; y++){
            printf("%c", block);
        }
        printf("  ");
        for (int r = 0; r < x + 1; r++){
            printf("%c", block);
        }
        printf("\n");
        
        
        
    }
    

    
    
}

File: Python Web - Problem Sets/pset6/mario.py
import cs50
import sys

def main():
    block = '#'
    space  = ' '
    intInput = 0
    while True:
        print("Please enter a number: ", end="")
        intInput = cs50.get_int()
        if intInput >= 0 and intInput <= 23:
            break
        else:
            continue
        
    for x in range(intInput):
        for t in range(intInput - x):
            print(space, end="")
        for y in range(x + 1):
            print(block, end="")
        print("  ", end="")
        for r in range(x + 1):
            print(block, end="")

        print("")
        
        
        
if __name__ == "__main__":
    main()